helloworld/notebooks/ListOfFiles.ipynb

```diff
@@ -0,0 +1,128 @@
+{
```

Comment:
why do we need `ListOfFiles.ipynb`?

Reply:
test notebook to make sure training data is mounted in the container path, will remove it

Reply:
removed